charts/camunda-platform-8.4/RELEASE-NOTES.md

```diff
@@ -1,51 +1,34 @@
 The changelog is automatically generated and it follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format.
 
-## [camunda-platform-8.6-11.3.0](https://github.com/camunda/camunda-platform-helm/releases/tag/camunda-platform-8.6-11.3.0) (2025-03-19)
-
-### Features
-
-- Add global.extraManifests support for injecting arbitrary YAML (#3050)
+## [camunda-platform-9.5.1](https://github.com/camunda/camunda-platform-helm/releases/tag/camunda-platform-9.5.1) (2025-05-08)
 
 ### Fixes
 
-- Changing helper function for identityURL (#3084)
+- Task 448 backport and refactor Camunda unit test refactor from 8.6 to 8.4 (#3302)
+- Add elasticsearch golden files (#3334)
+- Refactor 8.4 unit test connectors (#3354)
+- Refactor of optimize in 8.4 (#3361)
+- Refactor 8.4 unit test operate (#3362)
+- Refactored unit tests in 8.4 to new style in tasklist (#3360)
+- Refactored unit tests in 8.4 to new style in zeebe (#3359)
+- Refactored unit tests in 8.4 to new style in zeebe-gateway (#3358)
+- Refactor 8 4 unit test identity (#3356)
+- Refactor 8.4 unit test console (#3355)
+- Refactored unit tests in 8.4 to new style in web modeler (#3357)
+- Add apiVersion and kind to PersistentVolumeClaims (#2321)
 
 <!-- generated by git-cliff -->
 ### Release Info
 
```

Comment:
This is wrong. there should be version output here

Reply:
How did you get it back there?